src/main/java/seedu/duke/util/ExceptionMessage.java

```diff
@@ -51,8 +51,8 @@
             + "Please use 'help' to see our list of commands.\n";
     public static final String MESSAGE_INVALID_GRADE =
             "Your grade input isn't part of the NUS grading scheme. \n"
-            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F.\n"
-            + "Do not enter grades CS and CU - leave the module ungraded!\n";
+            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F, CS, CU. \n"
+            + "Grades have to be in uppercase not lower case. \n";
```

Comment:
Possible to turn user input into uppercase automatically?

Reply:
yes boss